components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -152,17 +152,21 @@ class PageWallet extends React.Component<Props, State> {
     return utils.convertBalance(balance.total.toString(), balance.rates)
   }
 
-  getGrants = () => {
-    const grants = this.props.rewardsData.walletInfo.grants
-    if (!grants) {
+  generatePromotions = () => {
+    const promotions = this.props.rewardsData.promotions
+    if (!promotions) {
       return []
     }
 
-    return grants.map((grant: Rewards.Grant) => {
+    let claimedPromotions = promotions.filter((promotion: Rewards.Promotion) => {
+      return promotion.status === 4 // PromotionStatus::FINISHED
```

Comment:
nit: TODO so a const doesn't get forgotten

Reply:
right now we can't use direct definition so I added reference for a number